File: algorithms/2114.maximum-number-of-words-found-in-sentences/solution.rs
impl Solution {
    pub fn most_words_found(sentences: Vec<String>) -> i32 {
        let mut max = 0i32;
        for s in sentences {
            let count = s.split_whitespace().collect::<Vec<&str>>().len();
            max = max.max(count as i32);
        }

        max
    }
}

File: algorithms/2114.maximum-number-of-words-found-in-sentences/solution.go
func mostWordsFound(sentences []string) int {
	max := 0

	for _, s := range sentences {
		count := wordCount(s)
		if count > max {
			max = count
		}
	}

	return max
}

func wordCount(s string) int {
	words := strings.Fields(s)
	return len(words)
}
